src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,649 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures aggregation expression performance.
+
+  While measuring performances, this workload collects numbers
+  for either the SBE or the classic engine aggregation expression
+  implementations, depending on environments that it runs on.
+
+  Numbers on the 'standalone-all-feature-flags' environment are for
+  the SBE aggregation expressions and numbers on the 'standalone' environment
+  for the classic aggregation expressions.
+
+Keywords:
+- aggregate
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 10
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 64
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          int: &integer {^RandomInt: {min: -100, max: 100}}
+          int2: *integer
+          negativeInt: {^RandomInt: {min: -1000, max: -1}}
+          posInt: {^RandomInt: {min: 1, max: 100}}
+          nullVal: null
+          string: &string {^RandomString: {length: 16}}
+          string2: *string
+          hello: "hellohellohellohellobye"
+          upper: "THISISANALLUPPERCASESTRING"
+          preDeterminedString: {^Choose: {from: ["one", "two", "three"], weights: [1, 1, 1]}}
+          array: &array [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          array2: *array
+          nestedObj: {int: *integer, array: *array}
+          double: &double {^RandomDouble: {distribution: normal, mean: 50, sigma: 13}}
+          doubleBounded: {^RandomDouble: {min: -1, max: 1}}
+          doubleBoundedNonZero: {^Choose: {from: [^RandomDouble: {min: -1, max: -0.000000001}, ^RandomDouble: {min: 0.000000001, max: 1}], weights: [1, 1]}}
+          date1: &date {^RandomDate: {min: "2021-01-01T00:00:02.000Z", max: "2054-01-01T00:00:10.000Z"}}
+          date2: *date
+          randomType: {^Choose: {from: [*integer, *double, *date, *string, *array, null], weights: [1, 1, 1, 1, 1, 1]}}
+
+# Phase 1: Ensure all data is synced to disk.
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+- ExpressionAbs:
```

Comment:
[nit] comment:

Phase 2-n: Run expression benchmarks

Or something like that

Reply:
Done.